tests/basic/misc_test.go

```diff
@@ -1157,3 +1161,102 @@ var _ = Describe("{AutoFSTrimReplAddWithNoPool0}", func() {
+
+		// Fetch random storage node and detach the disks attached to that node
+		var oldNodeIDtoMatch string
+		var newNodeIDtoMatch string
+		Step(fmt.Sprintf("detach disks attached from a random storage node %s and attach to non px node %s", randomStorageNode.Name, randNonPxNode.Name), func() {
+			// ToDo - Ensure the above selected node has volumes/apps running on it
+			// Store the Node ID of randomStorageNode for future use
+			oldNodeIDtoMatch = randomStorageNode.Id
+			// Stop PX on the node
+			err = k8sCore.AddLabelOnNode(randomStorageNode.Name, schedops.PXServiceLabelKey, "stop")
+			log.FailOnError(err, fmt.Sprintf("Failed to add label %s=stop on node %s", schedops.PXServiceLabelKey, randomStorageNode.Name))
+
+			err = Inst().N.MoveDisks(randomStorageNode, randNonPxNode)
+			log.FailOnError(err, fmt.Sprintf("Failed to move disks from node %s to node %s", randomStorageNode.Name, randNonPxNode.Name))
+
+			// Add PXEnabled false label to srcVM
+			err = k8sCore.AddLabelOnNode(randomStorageNode.Name, schedops.PXEnabledLabelKey, "false")
+			log.FailOnError(err, fmt.Sprintf("Failed to add label %s=stop on node %s", schedops.PXServiceLabelKey, randomStorageNode.Name))
+
+			// Start PX on the node
+			err = k8sCore.AddLabelOnNode(randNonPxNode.Name, schedops.PXEnabledLabelKey, "true")
+			log.FailOnError(err, fmt.Sprintf("Failed to add label %s=true on node %s", schedops.PXEnabledLabelKey, randNonPxNode.Name))
+			err = k8sCore.AddLabelOnNode(randNonPxNode.Name, schedops.PXServiceLabelKey, "start")
+			log.FailOnError(err, fmt.Sprintf("Failed to add label %s=start on node %s", schedops.PXServiceLabelKey, randNonPxNode.Name))
+			err = Inst().V.WaitForPxPodsToBeUp(randNonPxNode)
+			log.FailOnError(err, fmt.Sprintf("Failed to wait for PX pods to be up on node %s", randNonPxNode.Name))
+			// Refresh the driver endpoints
+			err = Inst().S.RefreshNodeRegistry()
+			log.FailOnError(err, "error refreshing node registry")
+			err = Inst().V.RefreshDriverEndpoints()
+			log.FailOnError(err, "error refreshing storage drive endpoints")
+			// Wait for the driver to be up on the node
+			randNonPxNode, err = node.GetNodeByName(randNonPxNode.Name)
+			if err != nil {
+				log.FailOnError(err, fmt.Sprintf("Failed to get node %s", randNonPxNode.Name))
+			}
+			err = Inst().V.WaitDriverUpOnNode(randNonPxNode, Inst().DriverStartTimeout)
+			dash.VerifyFatal(err, nil, "Validate volume is driver up")
+			// Verify the node ID is same as earlier stored Node ID
+			newNodeIDtoMatch = randNonPxNode.Id
+
+			if oldNodeIDtoMatch != newNodeIDtoMatch {
+				log.FailOnError(fmt.Errorf("node ID mismatch"), fmt.Sprintf("Node ID mismatch for node %s", randNonPxNode.Name))
+				dash.VerifyFatal(false, true, fmt.Sprintf("Node ID mismatch for node %s after moving the disks", randNonPxNode.Name))
+			} else {
+				log.Infof("Node ID matches for node %s [%s == %s]", randNonPxNode.Name, oldNodeIDtoMatch, newNodeIDtoMatch)
+			}
```

Comment:
How about? Instead of if else.

```go
dash.VerifyFatal(oldNodeIDtoMatch, newNodeIDtoMatch, fmt.Sprintf("Node ID mismatch for node %s after moving the disks", randNonPxNode.Name))
log.Infof("Node ID matches for node %s [%s == %s]", randNonPxNode.Name, oldNodeIDtoMatch, newNodeIDtoMatch)
```

Reply:
done